.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,14 @@
-## Explanation
+## Overview
+<!--
+READ ME FIRST: Please answer *both* questions below and check off every point from the essential checklist!.
```

Comment:
Nit: probably don't need the dot at the end?

Reply:
Removed!